.github/workflows/release-workflow.yml

```diff
@@ -65,41 +65,17 @@ jobs:
           draft: true
           prerelease: false
 
-      # Upload the release with .zip as asset
+      # Upload the release with .zip,.rpm and .deb as assets
       - name: Upload Release Asset
         id: upload-release-asset
-        uses: actions/upload-release-asset@v1.0.1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        with:
-          upload_url: ${{ steps.create_release.outputs.upload_url }}
-          asset_name: alerting.zip
-          asset_path: alerting-artifacts/*.zip
-          asset_content_type: application/zip
-
-      # Upload the release with .rpm as asset
-      - name: Upload Release Asset
-        id: upload-release-asset
-        uses: actions/upload-release-asset@v1.0.1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        with:
-          upload_url: ${{ steps.create_release.outputs.upload_url }}
-          asset_name: alerting.rpm
-          asset_path: alerting-artifacts/*.rpm
-          asset_content_type: application/zip
-
-      # Upload the release with .deb as asset
-      - name: Upload Release Asset
-        id: upload-release-asset
-        uses: actions/upload-release-asset@v1.0.1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        uses: softprops/action-gh-release@v1
```

Comment:
It looks like this action will default to using the tag as the release name for the name input:
https://github.com/softprops/action-gh-release#inputs

But, in the action previous to this one, we have `release_name: Release ${{ env.TAG_VERSION }}`, but there is also a `tag_name`.
Will this action be able to correctly find the release to upload the artifacts?

Reply:
I guess it should work if I add the following:

if: startsWith(github.ref, 'refs/tags/')
It will find the matching tag and then release accordingly